### PR TITLE
Add final events from 2018

### PR DESCRIPTION
**Pull request by mattj-io**
## Description
https://jira.mesosphere.com/browse/DCOS_OSS-2903
https://jira.mesosphere.com/browse/DCOS_OSS-2976
https://jira.mesosphere.com/browse/DCOS_OSS-2806

## Urgency
- [ ] Blocker 
- [ ] High
- [ x] Medium

## Requirements
- See the [contribution guidelines](https://github.com/dcos/dcos-website#contribution-workflow).
- Build content [locally](https://github.com/dcos/dcos-website#testing-your-updates-locally) and test for formatting/links.

### CODE REVIEW

**Comment by mattj-io** [comment]
Not that this will make much sense to you right now ;)

**Comment by yankcrime** [comment]
@mattj-io Given these events are in the past, do you still want comments on the copy?

**Comment by mattj-io** [comment]
The copy is usually the submitted abstract. Comment if you like though !

File: src/events/2018-09-20_ip_expo_nordics.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+category: Introduction
+description: Apache Mesos and DC/OS for deploying cloud native applications
+layout: event.jade
+collection: events
+lunr: true
+---
+
+# Abstract
+
+Data processing paradigms are undergoing a paradigm shift as we move more and more towards real time processing. Emerging software models such as the SMACK stack are at the forefront of this change, focused on a pipeline processing model, but are also introducing new levels of operational complexity in running multiple complex distributed systems such as Spark, Kafka and Cassandra. In this talk, I'll introduce both Apache Mesos and DC/OS as a solution to this growing problem, and describe the benefits are of running these new kinds of systems for emerging cloud native workloads.
```
Comment (yankcrime) [inline src/events/2018-09-20_ip_expo_nordics.md]: "Paradigm(s)" twice in the same sentence is a bit repetitive.

Comment (yankcrime) [inline src/events/2018-09-20_ip_expo_nordics.md]: A link to a definition of the SMACK stack would be handy.

Comment (yankcrime) [inline src/events/2018-09-20_ip_expo_nordics.md]: "[..] and describe the benefits are of" - needs a word adding or a word deleting...

Comment (yankcrime) [inline src/events/2018-09-20_ip_expo_nordics.md]: "Complexity" and "complex" in the same sentence.

**Pull request merged**